serviceWrappers/linux/kLiveController

```diff
@@ -57,7 +57,7 @@ loadNvm() {
 
 start() {
     [ -d ${LOG_DIR} ] || mkdir -p ${LOG_DIR}
-    loadNvm
+    #loadNvm
```

Comment:
Installing the official RPM/deb packages is better for Prod. too and it would take exactly 1 minute to change it but like I said before, their considerations are theirs and should not affect the OP packages. 

If @GElkayam wishes to continue to use NVM, he can. Here are several options:
- Use a different init script. Usually SaaS specific scripts go into either the 'saas-' repos or into ITProd's own repo. 

- We can have two sets of init scripts in the liveDVR repo, though personally, I think that's not a good approach.

- Alternatively, and perhaps, best, you can change the script so that:
```
KLIVE_CONTROLLER_PREFIX="/opt/kaltura/liveController/latest"
LOG_DIR="/var/log/liveController"
PID_DIR="/var/run"
```
is removed and more specific ENV vars like LIVEDVR_LOG_DIR and LIVEDVR_RUN_DIR are used and then I can set these and KLIVE_CONTROLLER_PREFIX in say /etc/default/livedvr [Debian/Ubuntu] and /etc/sysconfig/livedvr [RHEL/CentOS] and IT Prod. can do the same on their system.
and thr loadNVM function can still check for:
```
 if [ -s "${NVM_DIR}/nvm.sh" ]; then
```
and source it if it exists but not exit if it does not. Once again, requiring it is wrong since your code can work with any valid NodeJS of version 6 and higher, it does not depend on anything that is provided by NVM exclusively.